Fix/full url in queue

ben-en opened the pull request:
This this fix uses the full domain and port in the url to add or remove files from the bin to avoid triggering a mixed content error..

Commit:
added requirement for newer bottle-utils

Commit:
use full url in queue list

Commit:
use newly released version of bottle-utils

goodbadwolf commented on conf/requirements.txt:
```diff
@@ -3,7 +3,7 @@ mako
 cssmin==0.2.0
 sqlize
-bottle-utils==0.9
+bottle-utils==2.0.dev1
```
Please update it to use 2.0 or `>=2.0`

Commit:
provide full scheme in URL

Commit:
get scheme from config instead of bottle

    We chose to put the scheme in the config because we are using a reverse
    proxy. The app itself is not https, but we are using https.
